### PR TITLE
Home page with suggested fixed

### PR DESCRIPTION
**Pull request by Badrhoul**

### CODE REVIEW

File: client/app/(tabs)/homeScreen.tsx [hunk 1]
```diff
@@ -1,29 +1,78 @@
...
 const homeScreen = () => {
-  const { setUser, setSelectedProfile, setProfiles } = useDataContext();
+  const screenWidth = Dimensions.get('window').width;
+  const squareSize = screenWidth / 2.5;
+  const router = useRouter()
+
+  // const [stories, setStories] = useState<StoryType[]>([]); // actual state
+
+  const { allStories, setAllStories, setSelectedStory, selectedProfile } = useDataContext();
   // TODO: remove this one authentication is implemented
```
Comment (harrietmorris) [inline client/app/(tabs)/homeScreen.tsx:19]: it's been removed?

File: client/services/apiService.ts [hunk 2]
```diff
@@ -1,15 +1,15 @@
 import { ProfileType, StoryType, UserType } from '../types';
-import axios, { AxiosResponse } from 'axios';
+import axios, { AxiosResponse, isCancel, AxiosError } from 'axios';
 
-//const BASE_URL = 'http://localhost:3000'; //this may need to change to IP address
-const BASE_URL = 'http://10.0.2.2:3000'; //this is the URL used for android simulator
```
Comment (harrietmorris) [inline client/services/apiService.ts:5]: please don't delete this

File: client/app/(tabs)/homeScreen.tsx [hunk 3]
```diff
@@ -1,29 +1,78 @@
...
+      if (selectedProfile) {
+        const storiesByProfile = await getAllStoriesByProfile(selectedProfile.id);
+        setAllStories(storiesByProfile)
+      }
+    };
+
+    renderStoriesbyProfile();
+  }, [allStories]);
+
+  //should take an argument 'storyId"
```
Comment (sebastiandve) [inline client/app/(tabs)/homeScreen.tsx:33]: whats up with this comment?

File: client/services/apiService.ts [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
+const BASE_URL = 'http://localhost:3000'; //this may need to change to IP address
 
 export const getUser = async (id: number): Promise<UserType> => {
   try {
     const response: AxiosResponse<UserType> = await axios.get(`${BASE_URL}/users/${id}`);
+    console.log('getting user:', response.data);
```
Comment (sebastiandve) [inline client/services/apiService.ts:9]: delete this log

File: client/services/apiService.ts [hunk 5]
```diff
@@ -1,15 +1,15 @@
...
     return response.data;
   } catch (e) {
-    console.error('error getting user', e);
+    console.log('error getting user', e);
```
Comment (sebastiandve) [inline client/services/apiService.ts:12]: this should be an error

**Review by sebastiandve — CHANGES_REQUESTED** [review]
same requests as before

File: client/services/apiService.ts (hunk 2 above)
Comment (sebastiandve) [reply client/services/apiService.ts:5]: agree

**Review by sebastiandve — APPROVED** [review]
LGTM

**Pull request merged**